c2cgeoportal/forms.py

```diff
@@ -264,7 +264,10 @@ def render_organisational_item(self, item, depth):
         return result
 
     def is_checked(self, item, final_item):
-        return self._is_checked(final_item.id)
+        return final_item.id in self.values
```

Comment:
Minor but: With a method that is named "is"_somethings, I expect a boolean value. Now, this seems return a number (id)...